cmd/bad-key-revoker/main.go

```diff
@@ -29,6 +29,12 @@ import (
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
+const blockedKeysGaugeLimit = 1000
+
+var keysToProcess = prometheus.NewGauge(prometheus.GaugeOpts{
+	Name: "bad_keys_processed",
```

Comment:
Copy-paste from below that didn't get updated:
```suggestion
	Name: "bad_keys_to_process",
```

Reply:
Oof, thanks!